easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.0.1'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'opt': True, 'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('KEALib', '1.4.4'),
+    ('netCDF', '4.3.3.1'),
+    ('HDF5', '1.8.16'),
+    ('expat', '2.1.0'),
+    ('libxml2', '2.9.3', '-Python-2.7.11'),
```

Comment:
any particular reason you picked a libxml2 that was built with Python support enabled?

doesn't seem to be needed?